rosidl_typesupport_connext_c/package.xml

```diff
@@ -10,19 +10,20 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
   <buildtool_depend>connext_cmake_module</buildtool_depend>
   <buildtool_depend>rosidl_cmake</buildtool_depend>
-  <buildtool_depend>rosidl_generator_c</buildtool_depend>
   <buildtool_depend>rosidl_typesupport_connext_cpp</buildtool_depend>
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
   <buildtool_export_depend>connext_cmake_module</buildtool_export_depend>
   <buildtool_export_depend>rcutils</buildtool_export_depend>
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
-  <buildtool_export_depend>rosidl_generator_c</buildtool_export_depend>
+  <buildtool_export_depend>rosidl_runtime_c</buildtool_export_depend>
   <buildtool_export_depend>rosidl_generator_dds_idl</buildtool_export_depend>
   <buildtool_export_depend>rosidl_typesupport_connext_cpp</buildtool_export_depend>
 
   <build_export_depend>rmw</build_export_depend>
 
+  <build_depend>rosidl_runtime_c</build_depend>
```

Comment:
Please follow the order in the schema: https://github.com/ros-infrastructure/rep/blob/d09bc91ebd4ff963b09bf026573ae2ea9fd0987f/xsd/package_format3.xsd#L93-L102.

Reply:
Should I move this too? or in other PR?

```
<build_export_depend>rmw</build_export_depend>
```

Reply:
You can move both in this PR.